Gemfile.lock

```diff
@@ -315,76 +316,78 @@ GEM
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
+    domain_name (0.6.20240107)
+    doorkeeper (5.6.8)
       railties (>= 5)
-    dotenv (2.2.1)
+    dotenv (2.8.1)
```

Comment:
### [dotenv](https://github.com/bkeepers/dotenv)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.2.1 → 2.8.1

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/dotenv/2.2.1/2.8.1)_]





<details>
<summary>Commits</summary>

A change of **183** commits. See the full changes on [the compare page](https://github.com/bkeepers/dotenv/compare/v2.2.1...v2.8.1).

These are the first 10 commits:
- (697865b) [Skip over lines that don't set an envvars](https://github.com/bkeepers/dotenv/commit/697865b86bcc2c4cc33c09dd516d02c1ba28e5a8)
- (6e885a1) [Changes script type](https://github.com/bkeepers/dotenv/commit/6e885a18bc3193283b7ae4b90abcfa37768352a5)
- (a47020f) [Merge pull request #298 from edersonbadeca/change-read-me](https://github.com/bkeepers/dotenv/commit/a47020f6c414e0a577680b324e61876a690d2200)
- (3d0b424) [Document the -f flag](https://github.com/bkeepers/dotenv/commit/3d0b4245ceb1f88ca6c09274fe9f5aa60e55d15f)
- (a689e6a) [Existing variable takes precendence when performing substitu…](https://github.com/bkeepers/dotenv/commit/a689e6ae68b99b88626e5fe9e675dd17b730167b)
- (a7fd9b5) [CI against Ruby 2.5.0](https://github.com/bkeepers/dotenv/commit/a7fd9b59306b362f0183ccd02151d65a048266fc)
- (c7d6342) [Parse multi-line values properly (e.g. Heroku config export)](https://github.com/bkeepers/dotenv/commit/c7d634251f42b9d6f80f888e469a52f727c5a9a5)
- (c9c82b0) [Fix original regex to escape slash](https://github.com/bkeepers/dotenv/commit/c9c82b025014fcd1dc3484ce175acbd429ae4a60)
- (694c566) [Match multiple lines](https://github.com/bkeepers/dotenv/commit/694c566da91acf9768daf6ecd2334f3572ba0de2)
- (3bb0357) [Add tests for parsing multi-line values](https://github.com/bkeepers/dotenv/commit/3bb0357a07a7e76d485733e18a71f03aeddce32b)


</details>